src/title/TitleStream.js

```diff
@@ -69,17 +69,19 @@ const TitleStream = (props: Props) => {
 
   return (
     <TouchableWithoutFeedback
-      onLongPress={() => {
-        if (isTopicNarrow(narrow)) {
-          showHeaderActionSheet({
-            showActionSheetWithOptions,
-            callbacks: { dispatch, _ },
-            backgroundData,
-            stream: stream.name,
-            topic: topicOfNarrow(narrow),
-          });
-        }
-      }}
+      onLongPress={
+        isTopicNarrow(narrow)
+          ? () => {
+              showHeaderActionSheet({
+                showActionSheetWithOptions,
+                callbacks: { dispatch, _ },
+                backgroundData,
+                stream: stream.name,
+                topic: topicOfNarrow(narrow),
+              });
+            }
+          : undefined
```

Comment:
Mm, yeah; that's a good fix for the potential a11y issue here; hopefully `TouchableWithoutFeedback` can tell the difference between `undefined` and a function that does nothing, and do the right thing for a11y.

But let's squash the fix into the same commit that introduced the potential issue, I think.